JitsiMeetJS.js

```diff
@@ -42,6 +43,9 @@ var LibJitsiMeet = {
 
     version: '{#COMMIT_HASH#}',
 
+    // expose JitsiTrackError this way to give library consumers to do checks
+    // like if (error instanceof JitsiMeetJS.JitsiTrackError) { }
+    JitsiTrackError: JitsiTrackError,
```

Comment:
@tsareg I think we can add `LibJitsiMeet.errorTypes` map where you can expose the constructor. That way the users of the library are going to access it with `JitsiMeetJS.errorTypes.JitsiTrackError`. WDYT?

If you decide to do that trough different PR could you please remove this change from this PR?